src/container/internal.tsx

```diff
@@ -37,6 +37,7 @@ export default function InternalContainer({
   variant = 'default',
   disableHeaderPaddings = false,
   disableContentPaddings = false,
+  fitHeight,
```

Comment:
```suggestion
  fitHeight = false,
```

so that the documenter could extract the default value and show it on the website

Reply:
We already have this code on documentation website end

```
if (item.type === 'boolean') {
  return <CodeBlock code="false" />;
}
```

not needed to do this manually